src/coord/src/catalog.rs

```diff
@@ -208,6 +218,9 @@ impl CatalogItem {
             CatalogItem::Sink(_) => "sink",
             CatalogItem::View(_) => "view",
             CatalogItem::Index(_) => "index",
+            CatalogItem::Type(t) => match t {
+                Type::Map { .. } => "map type",
+            },
```

Comment:
I think this should just be "type" for all types of types, to match the other items.